14.0/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM debian:buster
+MAINTAINER Camptocamp
+
+# create the working directory and a place to set the logs (if wanted)
+RUN mkdir -p /odoo /var/log/odoo
+
+COPY ./base_requirements.txt /odoo
+COPY ./install /install
+
+# Moved because there was a bug while installing `odoo-autodiscover`. There is
+# an accent in the contributor name
+ENV LANG=C.UTF-8 \
+    LC_ALL=C.UTF-8
+
+# build and dev packages
+ENV BUILD_PACKAGE \
+    build-essential \
+    gcc \
+    python3.7-dev \
+    libevent-dev \
+    libfreetype6-dev \
+    libxml2-dev \
+    libxslt1-dev \
+    libsasl2-dev \
+    libldap2-dev \
+    libssl-dev \
+    libjpeg-dev \
+    libpng-dev \
+    zlib1g-dev \
+    git
+
+# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
+RUN set -x; \
+        sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list' \
+        && sh -c /install/package_odoo_13.0.sh \
```

Comment:
Shouldn't that script be updated for v14 ?

Reply:
Yes, you're right.

I was unsure there was anything else to be changed so I just kept it.
IMO i can rename it and update the build for V13. Do you agree?

Reply:
I think it's good to rename it to not confuse.
We can just copy it, I have use the image 13 to build v14 and it seems to work well.

Reply:
@vrenaville @TDu I changed the name of the script according to your reviews and what has been previously done.